CONTRIBUTING.md

```diff
@@ -2,29 +2,57 @@
+If any part of the operator-lifecycle-manager project has bugs or documentation mistakes, please let us know by [opening an issue][operator-olm-issue]. We treat bugs very seriously and believe no issue is too small. Before creating a bug report, please check that an issue reporting the same problem does not already exist. If you aren't sure if the observed behavior is a bug, please reach out to the OLM team using the links provided under [communication][communication].
+
+To make the bug report accurate and easy to understand, please try to create bug reports that are:
+
+- Specific. Include as much detail as possible: which version, what environment, what configuration, etc.
+
+- Reproducible. Include the steps to reproduce the problem. We understand some issues might be hard to reproduce, please include the steps that might lead to the problem.
+
+- Isolated. Please try to isolate and reproduce the bug with minimum dependencies. It would significantly slow down the speed to fix a bug if too many dependencies are involved in a bug report. Debugging external systems that rely on operator-lifecycle-manager is out of scope, but we are happy to provide guidance in the right direction or help with using operator-lifecycle-manager itself.
+
+- Unique. Do not duplicate existing bug report.
+
+- Scoped. One bug per report. Do not follow up with another bug inside one report.
+
+It may be worthwhile to read [Elika Etemad’s article on filing good bug reports][filing-good-bugs] before creating a bug report.
+
+We might ask for further information to locate a bug. A duplicated bug report will be closed.
 
 ## Contribution flow
 
 This is a rough outline of what a contributor's workflow looks like:
 
 - Create a topic branch from where to base the contribution. This is usually master.
-- Make commits of logical units.
-- Make sure commit messages are in the proper format (see below).
+- Make commits of logical units using the `--signoff` flag.
 - Push changes in a topic branch to a personal fork of the repository.
 - Submit a pull request to operator-framework/operator-lifecycle-manager.
-- The PR must receive a LGTM from two maintainers found in the MAINTAINERS file.
+- The pull request must then be reviewed by members of the operator framework team. Please notify the OLM Development team when you create a pull request using one of the links provided under [communication][communication]. As your PR is reviewed, please attempt to keep any design discussions within the pull request. If you have responded to all the messages on your PR and are concerned that it might have been forgotten, please ping us once again using one of the links provided under  [communication][communication]. Your PR will merge once:
```

Comment:
small nit:

```suggestion
- The pull request must then be reviewed by members of the operator framework team. Please notify the OLM Development team when you create a pull request using one of the links provided under [communication][communication]. As your PR is reviewed, please attempt to keep any design discussions within the pull request. If you have responded to all the messages on your PR and are concerned that it might have been forgotten, please ping us once again using one of the links provided under [communication][communication]. Your PR will merge once:
```